devbox.lock

```diff
@@ -0,0 +1,285 @@
+      "resolved": "github:NixOS/nixpkgs/ec04772e7516b6d58d98b491e68b329b7558b14d#gojq",
+      "source": "devbox-search",
+      "version": "0.12.14",
+      "systems": {
+        "aarch64-darwin": {
+          "store_path": "/nix/store/1jfjdjbbgm1f6pg74kq5421qz301pkcx-gojq-0.12.14"
+        },
+        "aarch64-linux": {
+          "store_path": "/nix/store/7c0xnfkjmpzc3r38zbh22zcm0rw404n5-gojq-0.12.14"
+        },
+        "x86_64-darwin": {
+          "store_path": "/nix/store/ymbbxg162p42sj59rvz7spkqk7pg05dj-gojq-0.12.14"
+        },
+        "x86_64-linux": {
+          "store_path": "/nix/store/gi0crp32w0gqyc8zd0xqdif6f04yy9jr-gojq-0.12.14"
+        }
+      }
+    },
+    "goreleaser@latest": {
+      "last_modified": "2023-11-17T14:14:56Z",
+      "resolved": "github:NixOS/nixpkgs/a71323f68d4377d12c04a5410e214495ec598d4c#goreleaser",
+      "source": "devbox-search",
+      "version": "1.22.1",
+      "systems": {
+        "aarch64-darwin": {
+          "store_path": "/nix/store/z6kyp206dx7m592fnzlqy5ibcn9qkb9j-goreleaser-1.22.1"
+        },
+        "aarch64-linux": {
+          "store_path": "/nix/store/vl7f3w9cm6ib9b3iz7bjd6s825rrxkmf-goreleaser-1.22.1"
+        },
+        "x86_64-darwin": {
+          "store_path": "/nix/store/bggx5xqyla8zpss4sv71p350ajd70f89-goreleaser-1.22.1"
+        },
+        "x86_64-linux": {
+          "store_path": "/nix/store/cgzhkcv3wvcd2zlblv4y0vhjvw479467-goreleaser-1.22.1"
+        }
+      }
+    },
+    "gotestsum@latest": {
+      "last_modified": "2023-11-17T14:14:56Z",
+      "resolved": "github:NixOS/nixpkgs/a71323f68d4377d12c04a5410e214495ec598d4c#gotestsum",
+      "source": "devbox-search",
+      "version": "1.10.1",
+      "systems": {
+        "aarch64-darwin": {
+          "store_path": "/nix/store/5skpsrl0cdr2s6yaqfymx575qw3n19nr-gotestsum-1.10.1"
+        },
+        "aarch64-linux": {
+          "store_path": "/nix/store/9xgdrdv18x2hnismc65kz5vcgbv3850a-gotestsum-1.10.1"
+        },
+        "x86_64-darwin": {
+          "store_path": "/nix/store/g7pf4yxmf0wvxw7bwaxsfyd2pjwry6ib-gotestsum-1.10.1"
+        },
+        "x86_64-linux": {
+          "store_path": "/nix/store/bjrv592a532c4nh40angvr3x4h574ff4-gotestsum-1.10.1"
+        }
+      }
+    },
+    "kind@latest": {
+      "last_modified": "2023-11-17T14:14:56Z",
+      "resolved": "github:NixOS/nixpkgs/a71323f68d4377d12c04a5410e214495ec598d4c#kind",
+      "source": "devbox-search",
+      "version": "0.20.0",
+      "systems": {
+        "aarch64-darwin": {
+          "store_path": "/nix/store/h63plvmjv7cb3i5ggqba1b2ac06sdvi5-kind-0.20.0"
+        },
+        "aarch64-linux": {
+          "store_path": "/nix/store/dwqcnipd3rj0kspwjvxdcm32sfi5pmdx-kind-0.20.0"
+        },
+        "x86_64-darwin": {
+          "store_path": "/nix/store/ip9nkl00d91jmqik7cmcbra8px7lvwyj-kind-0.20.0"
+        },
+        "x86_64-linux": {
+          "store_path": "/nix/store/5g25hx32zh9fkvc7sznhlv5v9yzsgba3-kind-0.20.0"
+        }
+      }
+    },
```

Comment:
I think this is problematic particularly in context of our CI system. Our e2e CI is running jobs on a Kubernetes cluster where the nodes are running on top EL7. EL7 has an ancient kernel which doesn't play well with newer cGroups APIs that newer versions of kind rely on. My guess is you're likely hitting this which is causing your e2es to fail.

Reply:
so it seems like i was able to start a cluster just fine with kind. The issue instead seems to be with creating a new workload cluster from the management cluster that was provisioned with clusterctl